motionplan/planManager.go

```diff
@@ -905,6 +905,17 @@ func (pm *planManager) planRelativeWaypoint(ctx context.Context, request *PlanRe
 		return nil, err
 	}
 
+	if opt.profile == PositionOnlyMotionProfile {
```

Comment:
@biotinker let me know your thoughts on this. 
Business logic of caring for position only mode is not in the algorithm, however I did need to introduce a new field to plannerOpts.